src/laravel/pagseguro/Sender/Sender.php

```diff
@@ -142,11 +142,11 @@ public function setName($name)
      */
     public function setPhone($phone)
     {
-        if ($phone === null) {
-            $phone = [];
+        if (!is_null($phone)) {
+            $this->phone = Phone::factory($phone);
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method setPhone uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/michaeldouglas010790/laravel-pagseguro/pullRequest?prid=5029546)